### PR TITLE
LPS-108860 Traffic sources

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by andresfulla** [comment]
Reviewing! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by andresfulla** [comment]
Sent here https://github.com/brianchandotcom/liferay-portal/pull/86308

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -24,10 +22,12 @@ sync-to-analytics-cloud=Sync to Liferay Analytics Cloud
 sync-to-analytics-cloud-help=In order to view Content Performance, your site has to be synced with Liferay Analytics Cloud.
 this-number-refers-to-the-total-number-of-reads-since-the-content-was-published=This number refers to the total number of reads since the content was published.
```
Comment (andresfulla) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties]: this-keys-are-going-to-be-so-much-fun! 😂

**Comment by dgarciasarai** [comment]
Pull request submitted to https://github.com/darquesdev/liferay-portal/pull/184. [See changes here.](https://github.com/dgarciasarai/liferay-portal/compare/93b6ca3b1847~1...93b6ca3b1847) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**